src/cli.ts

```diff
@@ -298,7 +298,7 @@ export async function initCtxJsonFromOptions(options: OptionValues): Promise<voi
 //////////////////////////////////////////////////////////////////////////////////////////
 // Network info
 
-function logAddressInfo(ctx: Context) {
+export function logAddressInfo(ctx: Context) {

```

Comment:
please add documentation for all the exported function, interfaces, constants and enums